PyIRI/main_library.py

```diff
@@ -10,7 +10,8 @@
 ----------
 Forsythe et al. (2023), PyIRI: Whole-Globe Approach to the
 International Reference Ionosphere Modeling Implemented in Python,
-Space Weather.
+Space Weather, ESS Open Archive, September 28, 2023,
+doi:10.22541/essoar.169592556.61105365/v1.
```

Comment:
```suggestion
doi: 10.22541/essoar.169592556.61105365/v1.
```
The Wiley DOIs don't resolve in the document link tests.  Try adding a space to stop the link test from occurring.